src/vs/editor/contrib/inlineCompletions/browser/model/inlineCompletionsModel.ts

```diff
@@ -220,8 +222,10 @@ export class InlineCompletionsModel extends Disposable {
 			const userJumpedToActiveCompletion = this._jumpedToId.map(jumpedTo => !!jumpedTo && jumpedTo === this._inlineCompletionItems.get()?.inlineEdit?.semanticId);
 
 			const providers = changeSummary.provider ? [changeSummary.provider] : this._languageFeaturesService.inlineCompletionsProvider.all(this.textModel);
+			const suppressedProviderGroupIds = this._suppressedInlineCompletionGroupIds.get();
+			const availableProviders = providers.filter(provider => !(provider.groupId && suppressedProviderGroupIds.has(provider.groupId)));
```

Comment:
To clarify, `groupId` is the extension ID. In the extension host `groupId: extensionId,` is set